perun-cli/getFacilitiesByHostName

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use Getopt::Long qw(:config no_ignore_case);
+use Text::ASCIITable;
+use Perun::Agent;
+use Perun::Common qw(printMessage getSortingFunction printTable);
+#use Data::Dumper;
+
+sub help {
+	return qq{
+	Lists Facilities where host is assigned.
+	--------------------------
+	Available options:
+	--hostname     | -H hostname
+	--orderByName  | -n order by name
+	--batch        | -b batch
+	--help         | -h prints this help
+	};
+}
+
+my ($hostname, $batch, $sortingFunction);
+
+GetOptions ("help|h"   => sub {
+		print help();
+		exit 0;
+	},
+	"batch|b"          => \$batch,
+	"orderByName|n"    => sub { $sortingFunction = getSortingFunction("getName", 1) },
+	"hostname|H=s"   => \$hostname
+) or die help();
+
+# Check options
+unless (defined $hostname) { die "ERROR: hostname is required \n";}
+unless (defined $sortingFunction) { $sortingFunction = getSortingFunction("getName", 1); }
+
+my $agent = Perun::Agent->new();
+my $facilitiesAgent = $agent->getFacilitiesAgent;
+
+my @facilities;
+FAC: @facilities = $facilitiesAgent->getFacilitiesByHostName( hostname => $hostname );
+unless (@facilities) {
+	printMessage "No facilities found for $hostname", $batch;
+} else {
+	#output
+	printTable($sortingFunction, @facilities);
+}	
+
+#zjisteni aliasu
+my $row;
+open (ALIAS,"host $hostname |") or die "system command host is not available\n";
```

Comment:
I don't think we need any of this. We want to call existing API method, which will find host by its hostname (i believe it is unique in perun) and it will return list of facilities (simply because we allow adding same host to multiple facilities). So in most cases it will simply return table with single facility.
There shouldn't be necessary any logic to guess "facility name" based on user provided / arbitrary "hostname". Admin should know, for which host he wants to get facility, since next step is to delete it (but it requires knowledge of facility name/id at the time).